### PR TITLE
chore: add .editorconfig

### PR DESCRIPTION
**Pull request by kvdb**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+max_line_length = 100
+trim_trailing_whitespace = true
```
Comment (joggienl) [inline .editorconfig:10]: trimming whitespace on markdown is not a good idea, perhaps add a special markdown section below (for example, you can use whitespace to create a new line, pretty common in markdown).

Prettier is also able to format markdown files (we use that in other projects as well). We try to use 80 chars for line length there.

**Review by joggienl — CHANGES_REQUESTED** [review]
See inline comment

File: .editorconfig (hunk 1 above)
Comment (kvdb) [reply .editorconfig:10]: It's only trimming trailing whitespace, it's not removing empty lines, so it should be no problem at all.
Didn't know that we use 80 chars except 79 for python code. I don't mind

Comment (joggienl) [reply .editorconfig:10]: trimming the trailing whitespace _is_ the problem. For markdown that should be as is, and not trimmed ;-)

Comment (joggienl) [reply .editorconfig:10]: About the line lenght: at this point for its-docs I am happy with either.

Comment (kvdb) [reply .editorconfig:10]: In theory you might be right, but I only encounter trailing whitespace in practice when I forget to remove it at the end of a sentence. That why for me this editorconf would work. It was basically the reason for me to add it in the first place. Do you ever leave whitespace at the end of a line in markdown for good reason?

Comment (joggienl) [reply .editorconfig:10]: yes actually, it is a feature of the language: trailing spaces mean something and therefore it is not OK to trim them.

I often use it.

**Pull request merged**